README.md

```diff
@@ -6,9 +6,75 @@ This repository is a WIP for the creation of a command line tool plugin for kube
 
 - clone/download this repo to you system
 - run `make mayactl`
-- use the command `kubectl mayactl` to use the plugin !
+- use the command `kubectl mayactl <storage engine> [command]` to use the
+plugin !
 
 ## Usage
 
-commands that have been implemented currently :
-* kubectl mayactl cStor volume list
+commands that have been implemented currently & sample outputs:
+* `kubectl mayactl cStor volume list`
+
+```bash
+Namespace  Name                                      Status   Version     Capacity  StorageClass          Attached  Access Mode      Attached Node
+---------  ----                                      ------   -------     --------  ------------          --------  -----------      -------------
+openebs    pvc-cb978ab8-9045-4d40-abc5-98dfd4fd82fb  Healthy  master-dev  5Gi       cstor.csi.openebs.io  Attached  [ReadWriteOnce]  vanisingh
+openebs    pvc-e20c1212-1ef6-42c4-9638-0145fa3fb4f9  Healthy  master-dev  5Gi       N/A                   N/A       []               N/A
+```
+
+* `kubectl mayactl cStor volume describe --volname <volname>`
+```bash
+Volume Details :
+----------------
+Name            : pvc-cbe030cb-63ca-4dfd-ba57-7719a8c93fb2
+Access Mode     : [ReadWriteOnce]
```

Comment:
Why the access mode has `[]` enclosed container? Do you get it as an array of items?

Reply:
AccessMode  is an array of corev1.PersistentVolumeAccessMode ... ill print just the elements as a string .